### PR TITLE
Adding napari-trackpy

### PR DESCRIPTION
**Pull request by rhoitink**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by rhoitink** [comment]
@conda-forge-admin, please ping team

File: recipes/napari-trackpy/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  sha256: 0f4097cbd1ef5fa616285c4b52e462896a1ec994b119576153ca4ca89e9d6416
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.8
```
Comment (goanpeca) [inline recipes/napari-trackpy/meta.yaml:19]: Could we sort this so it is easier to update and read :) ?

    - pip
    - python >=3.8
    - setuptools >=42.0.0
    - setuptools-scm
    - wheel

File: recipes/napari-trackpy/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    - setuptools >=42.0.0
+    - wheel
+    - setuptools-scm
+    - pip
+  run:
+    - python >=3.8
```
Comment (goanpeca) [inline recipes/napari-trackpy/meta.yaml:25]: Same here:

    - magicgui
    - matplotlib-base
    - napari
    - napari-aicsimageio >=0.7.0
    - numpy
    - python >=3.8
    - qtpy
    - trackpy

File: recipes/napari-trackpy/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    - napari_trackpy
+  commands:
+    - pip list 
+    # - pip check # Fails due to aicsimageio dependencies
+  requires:
+    - pip
+    - pyqt
+
+about:
+  home: https://github.com/rhoitink/napari-trackpy
```
Comment (goanpeca) [inline recipes/napari-trackpy/meta.yaml:45]:
```suggestion
  home: https://pypi.org/project/napari-trackpy
```

File: recipes/napari-trackpy/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+  summary: Plugin to do trackpy particle tracking on microscopy data within napari
+  license: MIT
+  license_file: LICENSE
```
Comment (goanpeca) [inline recipes/napari-trackpy/meta.yaml:48]:
```suggestion
  license_file: LICENSE
  dev_url: https://github.com/rhoitink/napari-trackpy
```

File: recipes/napari-trackpy/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+extra:
+  recipe-maintainers:
+    - rhoitink
```
Comment (goanpeca) [inline recipes/napari-trackpy/meta.yaml:52]:
```suggestion
    - goanpeca
    - rhoitink
```

**Review by goanpeca — CHANGES_REQUESTED** [review]
Hi @rhoitink, thanks for the contribution. Added some comments :)

**Review by goanpeca — CHANGES_REQUESTED** [review]
Hi @rhoitink, thanks for the contribution. Added some comments :)

File: recipes/napari-trackpy/meta.yaml (hunk 2 above)
Comment (rhoitink) [reply recipes/napari-trackpy/meta.yaml]: Done!

File: recipes/napari-trackpy/meta.yaml (hunk 1 above)
Comment (rhoitink) [reply recipes/napari-trackpy/meta.yaml:20]: Done!

**Review by rhoitink — COMMENTED** [review]
Hi @goanpeca, thanks for looking at it so quickly! Good comments, I've processed all of them and also updated it to the latest version (0.2.1) at the same time. Could you have a look again?

**Review by goanpeca — APPROVED** [review]
Looks good 🚀

**Pull request merged**